demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -728,6 +766,17 @@ class ContentList extends React.Component {
                         <div className="drm-choice">
                           {generateDRMButtons()}
                         </div>
+                        { isCustomDRM ?
+                          <div>
+                            <TextInput
+                              ariaLabel="Key system (reverse domain name)"
```

Comment:
Maybe we should give an example, as it not may be clear for every user ?

Reply:
Key system - reverse domain name (e.g. org.w3.clearkey)

Reply:
Okayx